framework/db/oci/QueryBuilder.php

```diff
@@ -162,6 +162,27 @@ public function addForeignKey($name, $table, $columns, $refTable, $refColumns, $
     }
 
     /**
+     * For each return column a param will be added to $returnParams.
+     * They should have variables bound before executing the query.
+     * @inheritdoc
+     */
+    public function insertReturning($table, $columns, &$params, $returnColumns = null, &$returnParams = null)
+    {
+        $schema = $this->db->getSchema();
```

Comment:
I don't think it's possible to return bound params by any `query*` method.

There would have to be a private property in `Command` class to track and return such params instead of results. Is it a good idea to add them?